osidb/migrations/0086_migrate_flaw_flag_nist_cvss_validation.py

```diff
@@ -0,0 +1,101 @@
+"""
+Written manually on 2023-07-17
+
+Copy FlawMeta NIST_CVSS_VALIDATION into Flaw field nist_cvss_validation.
+
+To prevent out of memory issues, FlawMeta are iterated using .iterator()
+so that FlawMeta instances are not cached. To prevent further out of
+memory issues and runtime efficiency, the migration uses a generator,
+islice and bulk_create as recommended by
+https://docs.djangoproject.com/en/3.2/ref/models/querysets/#bulk-create
+
+This migration should happen in lockstep with adding bzimport for Flaw
+field nist_cvss_validation. This ensures that any future operations performed
+using the new field do not result in data loss. (For a hypothetical example,
+if the flag value was recorded only in FlawMeta but not in the Flaw field,
+bbsync based on the field would effectively delete the flag from bugzilla,
+and that would also delete the FlawMeta-based flag on next bzimport.)
```

Comment:
As we already discussed, this situation currently cannot happen, so this paragraph evokes an unrealistic issue (which imho is not good).

I am not against migrations, but having them included in the commit `Add nist_cvss_validation to Flaw BZImport.` seems inappropriate. The commit name says nothing about migrations. If we stick to the fact that we should make them together with BZImport at the latest, it also implies that we can make them before the BZImport. So a nitpick from my side is to have the following commits: 1. migration, 2. bzimport, 3. bbsync, 4. api.

To be precise, migrations should be done together with a new model (if we want to do them now and not at the end of Basic Workflow).

Reply:
Also, `this situation currently cannot happen` is not applicable to this PR and this migration.

The `situation currently cannot happen` that we did discuss was about submitting PRs that enable writable API in the timeframe before enabling write access on the production instance.

This migration addresses a different problem that would happen on the read-only production instance. The problem and resolution is described using a hypothetical example because that's IMO the shortest explanation possible.

Reply:
After a 1:1, I'll delete the comment.

Reply:
Modified based on our 1:1.